.github/workflows/e2e-main.yaml

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (C) 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: e2e-tests-main
+
+on: 
+  push:
+    branches: [main]
+    
+  workflow_dispatch:
+    inputs:
+      fork:
+        default: 'containers'
```

Comment:
Someone may have a fork that may be called differently from podman-desktop so wouldn't it better to have fork be the org and the repo name (containers/podman-desktop) and use it accordingly

Reply:
Like this? org+repositoryName for a git repo to checkout?